src/script/conversation/ConversationRepository.ts

```diff
@@ -2354,14 +2406,14 @@ export class ConversationRepository {
       });
   }
 
-  _createTextProto(
-    messageId,
-    textMessage,
-    mentionEntities,
-    quoteEntity,
-    linkPreviews,
-    expectsReadConfirmation,
-    legalHoldStatus,
+  private _createTextProto(
+    messageId: string,
+    textMessage: string,
+    mentionEntities: MentionEntity[],
+    quoteEntity: QuoteEntity,
+    linkPreviews: any[],
```

Comment:
type for this should be LinkPreview[] and can be imported from `LinkPreviewRepository.ts`